src/Bridges/Constraint/bridges/inequality_to_complements.jl

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) 2017: Miles Lubin and contributors
+# Copyright (c) 2017: Google Inc.
+#
+# Use of this source code is governed by an MIT-style license that can be found
+# in the LICENSE.md file or at https://opensource.org/licenses/MIT.
+
+"""
+    InequalityToComplementsBridge{T,F,S,G} <: Bridges.Constraint.AbstractBridge
+
+`InequalityToComplementsBridge` implements the following reformulations:
+
+  * ``f(x) \\ge b`` into ``f(x) - b \\perp y \\ge 0``
```

Comment:
```suggestion
  * ``f(x) \\ge b`` into ``\\exists y`` such that ``f(x) - b \\perp y \\ge 0``
```

Reply:
Otherwise, `y` is not defined in this context

Reply:
Now it becomes `0 \le y \perp f(x) - b` which looks a bit less nice so I see why to put `y` in second place